Formula/cassandra-cpp-driver.rb

```diff
@@ -0,0 +1,37 @@
+class CassandraCppDriver < Formula
+  desc "DataStax C/C++ Driver for Apache Cassandra"
+  homepage "https://docs.datastax.com/en/developer/cpp-driver/latest"
+  url "https://github.com/datastax/cpp-driver/archive/2.15.0.tar.gz"
+  sha256 "8cccffc7f3b7a8edc51c5c4344fd6a60d5a7beec27a1a3fd52742003a2675cb9"
+  head "https://github.com/datastax/cpp-driver.git"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "cmake" => :build
+  depends_on "libtool" => :build
```

Comment:
```suggestion
```

Reply:
Wow, thanks for looking at this. You clearly have a lot more familiarity with this kind of stuff than I do. I just duplicated exactly what the documentation said to do for building on macOS.